tests/runtime_tests/test_mpa.py

```diff
@@ -0,0 +1,147 @@
+# Copyright 2025 Pasteur Labs. All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+"""Tests for the MPA (Model Performance Analytics) library."""
+
+import csv
+import json
+import os
+
+import pytest
+
+from tesseract_core.runtime import mpa
+
+
+class TestContextManagement:
```

Comment:
I don't mind these classes per se but this isn't used anywhere else in Tesseract Core, so I suggest you copy the test methods to global namespace instead.